exporter/v1_compatibility.go

```diff
@@ -815,9 +815,7 @@ func specialMetrics(ctx context.Context, client *mongo.Client, m bson.M, l *logr
 	}
 
 	if nodeType == typeMongod || nodeType == typeArbiter {
-		if engine, err := storageEngine(m); err != nil {
-			l.Errorf("cannot retrieve engine type: %s", err)
-		} else {
```

Comment:
do we know when we can't retrieve engine? it's important to not just hide an error, but hide it only for reason.